app/actions/remote/post.ts

```diff
@@ -552,3 +552,27 @@ export const togglePinPost = async (serverUrl: string, postId: string) => {
         return {error};
     }
 };
+
+export const deletePost = async (serverUrl: string, postId: string) => {
+    const database = DatabaseManager.serverDatabases[serverUrl]?.database;
+    if (!database) {
+        return {error: `${serverUrl} database not found`};
+    }
+
+    let client: Client;
+    try {
+        client = NetworkManager.getClient(serverUrl);
+    } catch (error) {
+        return {error};
+    }
+
+    try {
+        client.deletePost(postId);
```

Comment:
Should we handle any error on the client function? Or we expect anything that may go wrong to throw an exception? If so, should we at least await it?

Reply:
yup definitely await and do nothing if fails